.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8"
+  - "10"
```

Comment:
Node 10 won't be LTS until later this year. It's not really a big deal, but some things might change between the current version and LTS

Reply:
Okay so this is honestly just a work around since node 8 does not support conditional imports that i have in the server files. I'm not really sure how to get around it... I've looked at all the conversations on github issues and none of it seems to work.

Reply:
You don't have to make the import conditional. You could just move the mock data into the application and import it at the top. Then only use the mock data in certain environments. Does that make sense?

Reply:
I could do that. however, that means that those files will have to be included into the build as well or else our transpiler will get mad at us. And that is my reason for the conditional import

Reply:
That makes sense. If this isn't a temporary measure, there are ways to avoid keeping the mocking logic in the server files though. This is for testing, right? Well, you could either inject the server functions via props or move toward having these server functions dispatched by Redux. Either way you're allowing yourself a path to mock responses.